port/win32/omrsysinfo.c

```diff
@@ -773,6 +773,12 @@ omrsysinfo_get_memory_info(struct OMRPortLibrary *portLibrary, struct J9MemoryIn
 	return 0;
 }
 
+uint64_t
+omrsysinfo_get_addressable_physical_memory(struct OMRPortLibrary *portLibrary)
+{
+	return portLibrary->sysinfo_get_physical_memory(portLibrary);
```

Comment:
Are we not missing this check `if (OMRPORT_LIMIT_LIMITED == portLibrary->sysinfo_get_limit(portLibrary, OMRPORT_RESOURCE_ADDRESS_SPACE, &memoryLimit))`?

Reply:
I intentionally omitted the check as `sysinfo_get_limit()` on win32 always returns `OMRPORT_LIMIT_UNKNOWN_VALUE`.
Do you want me to add that check?